sphinx_needs/data.py

```diff
@@ -132,12 +138,14 @@ class CoreFieldParameters(TypedDict):
         "schema": {"type": ["string", "null"], "default": None},
         "show_in_layout": True,
         "exclude_external": True,
+        "allow_extend": True,
```

Comment:
Isn't this evaluated during `add_need` so needextend will not see the need if it is deleted?

Reply:
There is already a PR to completely remove this field, so it doesn't really matter: https://github.com/useblocks/sphinx-needs/pull/1347

Reply:
but removed anyway